sources/packages/api/src/services/student/student.service.ts

```diff
@@ -119,19 +128,20 @@ export class StudentService extends RecordDataModelService<Student> {
     externalEntityManager?: EntityManager,
     studentAccountApplicationId?: number,
   ): Promise<Student> {
+    // SIN to be saved and used for comparisons.
+    const studentSIN = removeWhiteSpaces(studentInfo.sinNumber);
+    const student =
+      (await this.getExistingStudentForAccountCreation(studentSIN, userInfo)) ??
```

Comment:
I have one clarification on the requirement. Currently by the existing logic, if anyone by chance gets to know the SIN of a student who is existing bcsc user, they can create a request with same SIN first name, last name and dob. And by doing so without even ministry acting upon it, the genuine user may lose access right?

Reply:
The student account creation happens at 2 moments.
1 - By a BSCS user, that is a user that had the identity validated by the BCSC. This user can provide wrong data but we trust him because we know his identity.
2 - The Ministry user, upon data validation the Ministry will allow the student creation.
Basic BCeIDs do not create student accounts without the Ministry's approval and we rely on that.